cmd/manager/main.go

```diff
@@ -93,14 +92,23 @@ func main() {
 		os.Exit(1)
 	}
 
-	logrus.Info("Registering Components")
+	logrus.Info("Registering Che Components Types")
 
 	// Setup Scheme for all resources
 	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
 		logrus.Error(err, "")
 		os.Exit(1)
 	}
 
+	logrus.Info("Registering Console Components Types")
```

Comment:
Did you test this under OpenShift 3.11 or OpenShift 4.1 ?
It seems to me that at least you should register the console objects and console scheme only when it is supported and available.
So I assume that at least here would be the right place: https://github.com/eclipse/che-operator/blob/master/pkg/controller/che/che_controller.go#L75

Reply:
Yes, I tested it on `OpenShift 3.11` and `OpenShift 4.2`
In my understanding, we are not registering CRD on openshift by invoking it, but configuring our k8s client to knows about such kind of objects.

This is still draft and I'm going to test it on Openshift 3.11, 4.1, 4.2 before making it ready to review.

Reply:
Could you still move your registration code here please, at least to be consistent with the rest of the code:

https://github.com/eclipse/che-operator/blob/9682f3448fe240c216aa22d9d3f56cc056b01494/pkg/controller/che/che_controller.go#L86

Reply:
> at least to be consistent with the rest of the code

Will do